psl-core/src/main/java/org/linqs/psl/model/rule/arithmetic/AbstractArithmeticRule.java

```diff
@@ -289,6 +290,11 @@ private void groundForNonSummation(Constant[] constants, Map<Variable, Integer>
         groundSingleNonSummationRule(constants, variableMap, atomManager, resources);
 
         results.addAll(resources.groundRules);
+        if (Options.CLI_VIZ.getBoolean()) {
```

Comment:
Diagram that paths for grounding in this class.
There is a more efficient choke that two of these collections can go in.